modules/mysql/variables.tf

```diff
@@ -115,6 +115,12 @@ variable "pricing_plan" {
   default     = "PER_USE"
 }
 
+variable "maintenance_window_enabled" {
+  description = "Whether to enable maintenance window setting for the instance. Replicas must have this set to false."
```

Comment:
Can you rename this slightly as:
"Whether to enable maintenance window setting for the instance. This does not apply to replicas."